specs/linux/selinux_events.table

```diff
@@ -0,0 +1,11 @@
+table_name("selinux_events")
+description("Track SELinux events.")
+schema([
+    Column("type", TEXT, "Event type"),
+    Column("msg", TEXT, "Message"),
```

Comment:
I will get back to a more-complete review later, but I don't want to forget things. I would recommend `message` as that is already used in `asl` and `user_events`. The `msg` shorthand is not used anywhere.

Reply:
Ah, my vote is still for `message` please :)